README.md

```diff
@@ -12,13 +12,19 @@ TypeScript support for ESLint. (This is still in the very early stages, so pleas
 
 You'll first need to install [ESLint](http://eslint.org):
 
-```
+```shellsession
 $ npm i eslint --save-dev
 ```
 
-Last, install `eslint-plugin-typescript`:
+Next, install `typescript` if you haven’t already:
 
+```shellsession
+$ npm i typescript@~3.1.1 --save-dev
```

Comment:
Does the tilde there really works? Just curious.
I found some problems in the past and now I'm using `npm i --save-dev --save-prefix=~ typescript@3.1.1`